custom-recipes/nlp-analysis-ontology-tagging/recipe.py

```diff
@@ -15,21 +14,20 @@
     lang, lang_column = settings.language, ""
 
 tagger = Tagger(
-    settings.text_input,
-    settings.ontology_input,
+    settings.text_input.get_dataframe(),
+    settings.ontology_input.get_dataframe(),
     settings.text_column,
     lang,
     lang_column,
     settings.tag_column,
     settings.category_column,
     settings.keyword_column,
     settings.lemmatization,
-    settings.case_sensitivity,
+    settings.case_insensitive,
     settings.unicode_normalization,
     settings.output_format,
 )
-tagging_formatter = TaggerFormatter(tagger)
-output_df = tagging_formatter.formatting_procedure()
+output_df = tagger.tag_and_format()
 
 output_dataset = get_output_names_for_role("tagged_documents")[0]
```

Comment:
You could do that directly in `settings` through DkuConfigLoadingOntologyTagging

Reply:
here : 8c4b1dc